cypress/e2e/tests/pages/global-settings/banners.spec.ts

```diff
@@ -111,12 +120,23 @@ describe('Banners', { testIsolation: 'off' }, () => {
 
       // Check in session
       bannersPage.banner().should('be.visible').then((el) => {
-        expect(el).to.have.attr('style').contains(`color: ${ settings.bannerTextColor.newRGB }`);
         expect(el).to.have.attr('style').contains(`background-color: ${ settings.bannerBackgroundColor.newRGB }`);
         expect(el).to.have.attr('style').contains(`text-align: ${ settings.textAlignment.new.toLowerCase() }`);
-        expect(el).to.have.attr('style').contains(`ext-decoration: ${ settings.textDecoration.toLowerCase() }`);
-        expect(el).to.have.attr('style').contains(`font-size: ${ settings.fontSize.new }`);
+        expect(el).to.have.attr('style').contains(`text-decoration: ${ settings.textDecoration.toLowerCase() }`);
```

Comment:
Unfortunately I can't move this check to be on the child. When checking computed properties `text-decoration` seems to vary from browser to browser and doesn't actually show underlined when I checked on the child.